recipe/meta.yaml

```diff
@@ -18,16 +18,13 @@ source:
 
 build:
   number: 2
-  # TODO: Add `linux_aarch64`, `linux_ppc64le`, and `win_64` once supported on CUDA 12 in conda-forge.
-  # xref: https://github.com/conda-forge/cuda-python-feedstock/issues/36
-  # xref: https://github.com/conda-forge/cuda-python-feedstock/issues/37
-  # xref: https://github.com/conda-forge/cuda-python-feedstock/issues/38
-  skip: true  # [win or osx or cuda_compiler_version != "12.0"]
+  skip: true  # [osx or cuda_compiler_version != "12.0" or py == 312]
```

Comment:
Should we add a comment about why Python 3.12 was skipped?

Reply:
```suggestion
  # Cython 0.29.x seems not to be fully compatible with Python 3.12, skipping for now.
  # This can be removed in a later CUDA Python that supports Cython 3.x.
  skip: true  # [osx or cuda_compiler_version != "12.0" or py == 312]
```